packages/vaadin-grid-pro/package.json

```diff
@@ -30,7 +30,7 @@
     "@vaadin/component-base": "^22.0.0-alpha6",
     "@vaadin/select": "^22.0.0-alpha6",
     "@vaadin/text-field": "^22.0.0-alpha6",
-    "@vaadin/vaadin-grid": "^22.0.0-alpha6",
+    "@vaadin/grid": "^22.0.0-alpha6",
```

Comment:
nit: let's sort dependencies alphabetically (so that grid would be after component-base).